errbot/plugin_manager.py

```diff
@@ -275,7 +275,11 @@ def _load_plugins_generic(
         feedback: Dict[Path, str],
     ):
         self._install_potential_package_dependencies(path, feedback)
-        plugfiles = path.glob("**/*." + extension)
+        
+        plugfiles = []
+        for ext in ("plug", "plugin"):
```

Comment:
This unfortunately would break the usage in https://github.com/errbotio/errbot/blob/3c3af1a00fc9ec047133f2a248eccec0e4a6d551/errbot/plugin_manager.py#L317-L338, as the parameters being passed include `plug` and `flow`. This change, unfortunately does not address that and include a new one named `plugin`.